### DISCUSSION TITLE
PBEM Public Playtesting

### DISCUSSION THREAD

**Discussion by hweimer** [raceintospace/raceintospace#693] [General]
Since the PBEM snake mode is now in master, I think it would be a good opportunity to test it under real-world conditions. Probably the easiest way is to attach the save games to this discussion [needs to be (g)zipped first to work with GitHub], as I have done with the first turn. Since there might be multiple persons interested in joining the test, I'd appreciate if you could choose your GitHub user name as the name of your save game.

Good game!

[PUBLICPBEM.SAV.gz](https://github.com/raceintospace/raceintospace/files/9703809/PUBLICPBEM.SAV.gz)

**Comment by peyre** [comment]
So you're thinking you and I should play a PBEM game to test it?  I'm game--I'll download, compile, and play tonight.

Come to think of it, I might be able to get started today.

Edit: I see we're playing the Basic Model.   Ah well.
